corehq/apps/app_manager/views/download.py

```diff
@@ -14,7 +14,7 @@
 
 from corehq import toggles
 from corehq.apps.app_manager.dbaccessors import get_all_built_app_ids_and_versions, get_app
-from corehq.apps.app_manager.decorators import safe_download, safe_cached_download
+from cgorehq.apps.app_manager.decorators import safe_download, safe_cached_download
```

Comment:
eep. `cgore`

Reply:
oh, thought that was the other way around